routers/repo/view.go

```diff
@@ -119,6 +120,21 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 	ctx.Data["LatestCommit"] = latestCommit
 	ctx.Data["LatestCommitVerification"] = models.ParseCommitWithSignature(latestCommit)
 	ctx.Data["LatestCommitUser"] = models.ValidateCommitWithEmail(latestCommit)
+	commit := models.SignCommitWithStatuses{
```

Comment:
Is there a reason you don't just use two local variables? The `SignCommit` field of `commit` never comes into play, so I think that just using variables would be simpler.

Reply:
done.